src/Components/SignIn.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+export default function SignIn() {
+  return (
+    <section className='signin-section'>
+      <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-6 text-black mx-auto signin">
+
+        <div class="px-5 ms-xl-4">
+          {/* <i class="fas fa-crow fa-2x me-3 pt-5 mt-xl-4" style={{"color": "#709085"}}></i> */}
+          <span class="h1 fw-bold mb-0">RESIN</span>
+        </div>
+
+        <div class="d-flex align-items-center h-custom-2 px-5 ms-xl-4 mt-5 pt-5 pt-xl-0 mt-xl-n5">
+
+          <form style={{"width": "23rem"}}>
+
+            <h3 class="fw-normal mb-3 pb-3" style={{"letter-spacing": "1px"}}>Log in</h3>
+
+            <div class="form-outline mb-4">
+              <input type="email" id="form2Example18" class="form-control form-control-lg" />
+              <label class="form-label" for="form2Example18">Email address</label>
```

Comment:
Since input & label are used multiple times in this form, take it out into separate functional component to reduce duplicity. Something like:
```
function input(props) {
      return (
            <div class="form-outline mb-4">
                <input type={props.email} id={props.id} class={props.class} />
                <label class={props.label} for={props.for}>Email address</label>
            </div>
      );
}
```